Reviewing Update portions of OS IDE docs (#1092)
ARMmbed/Handbook: docs/reference/technology/firmware_update.md

<span class="notes">**Note**: The default workflow of the development tools is intended only for development and testing purposes. It is not considered secure for production.</span>

Your development tool needs to use your Device Management account's API key to call the Pelion Device Management APIs. Once it has access to the APIs, it can generate a manifest and upload the manifest and binary to the server, and deliver the manifest to a device or group of devices. For more information, see [the update API documentation](https://www.pelion.com/docs/device-management/current/service-api-references/update-service.html).

<span class="notes">**Note**: The tools currently support the update process for Device Management Client, not Device Management Client Lite.</span>

**Tutorials**:

> - Try the firmware [update process on the Online Compiler](../guides/pelion-firmware-update/).

Reviewer: Query: Do the shortened links work if you're in another product's site?

Reviewer 2: No, of course not. They're relative.

Reviewer 2: (Or at least, it doesn't work when I try it)

Author: No, but both quick starts are hosted on the OS docs site. Are relative links still unsuitable since they're outside the docs set itself? 🤔

Reviewer 2: Try it. You have a build of that branch.

Author: Right. It worked for me, but went through a redirect because the link was to cloud.mbed.com. Odd. In any case, Pekka set me straight: it doesn't matter that the material is all hosted on the same domain; what matters is that the sources are in different repositories. Which makes total sense. So even though they're on the same product's (os.mbed.com) site, the quick connects still need absolute links. Mind blown.

Reviewer 2: No, it doesn't matter where we keep the source. It matters where we publish it. And you just said it worked. How did a relative link go to a redirect through Cloud?

> Your development tool needs to use your Device Management account's API key to call the Pelion Device Management APIs. Once it has access to the APIs, it can generate a manifest and upload the manifest and binary to the server, and deliver the manifest to a device or group of devices. For more information, see [the update API documentation](https://cloud.mbed.com/docs/latest/service-api-references/update-service.html).

Author: Should be a Pelion link
```suggestion
Your development tool needs to use your Device Management account's API key to call the Pelion Device Management APIs. Once it has access to the APIs, it can generate a manifest and upload the manifest and binary to the server, and deliver the manifest to a device or group of devices. For more information, see [the update API documentation](https://www.pelion.com/docs/device-management/current/service-api-references/update-service.html).
```

<span class="notes">**Note**: The tools currently support the update process for Device Management Client, not Device Management Client Lite.</span>

**Tutorials**:

> - Try the firmware [update process on the Online Compiler](https://cloud.mbed.com/guides/pelion-firmware-update).

Author: Needs updated link?

> - Try the firmware [update process on the Online Compiler](https://cloud.mbed.com/guides/pelion-firmware-update).

Author: ```suggestion
- Try the firmware [update process on the Online Compiler](../guides/pelion-firmware-update/).
```

>    <span class="notes">**Note:** You first need to [use the connecting quick guide to set up the original application](https://cloud.mbed.com/guides/connect-device-to-pelion).</span>

Author: ```suggestion
   <span class="notes">**Note:** You first need to [use the connecting quick guide to set up the original application](../guides/connect-device-to-pelion).</span>
```

Your development tool needs to use your Device Management account's API key to call the Pelion Device Management APIs. Once it has access to the APIs, it can generate a manifest and upload the manifest and binary to the server, and deliver the manifest to a device or group of devices. For more information, see [the update API documentation](https://cloud.mbed.com/docs/latest/service-api-references/update-service.html).

<span class="notes">**Note**: The tools currently support the update process for Device Management Client, not Device Management Client Lite.</span>

**Tutorials**:

> - Try the firmware [update process on the Online Compiler](https://cloud.mbed.com/guides/pelion-firmware-update). Note: you first need to [use the connecting quick guide to set up the original application](https://cloud.mbed.com/guides/connect-device-to-pelion).

Author: Should this be an actual note?

Reviewer: Yes, let's make it an actual note. 👍
